docker/test/integration/minifi/test/__init__.py

```diff
@@ -55,6 +55,9 @@ def __init__(self, output_validator):
         logging.info('Creating tmp test resource dir: %s', self.tmp_test_resources_dir)
         os.makedirs(self.tmp_test_resources_dir)
 
+        # Add resources
+        shutil.copytree("docker/test/integration/resources/kafka_broker/conf/certs", self.tmp_test_resources_dir + "/certs")
```

Comment:
This does not work for me with `make docker-verify`. I don't see how it could work, as we are normally standing in the build directory, but this is a path relative to the source directory.

Reply:
ah yea, sorry, usually I use ./DockerVerify.sh <version>. Do you think I should modify the relative path to suit with `make docker-verify` or find another way to get path?

Reply:
Since `make docker-verify` is, AFAIK, the standard way to run the docker tests, that should at least work, but generally the thing should work without depending on the current working directory (or, at least, only depend on it being the build directory). Keep in mind that it is completely supported to have out-of-tree builds, meaning that the build directory is not the subdirectory of the source directory, but a completely separate one.